### ISSUE TITLE
Allow iagl.digital and subdomains

### ISSUE THREAD

**Issue by Dynamic5912** [hagezi/dns-blocklists#2700]
### Which AdBlocker/DNS cloud service do you use?

AdGuard Home

### ControlD users

- [ ] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [ ] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Most Abused TLDs

### Which domain(s) should be unblocked?

iagl.digital
cdn.rewardsapp.iagl.digital
collect.rewardsapp.iagl.digital

There may be other subdomains too

### Why should the domain(s) be unblocked?

Required for British Airways Avios App to work.

**Comment by Dynamic5912** [comment]
This still appears to get blocked.

**Comment by hagezi** [comment]
No:
https://github.com/hagezi/dns-blocklists/blob/479cf57de55c5332af63e68d3cfe78eabbaf6412/adblock/spam-tlds.txt#L44

**Comment by Dynamic5912** [comment]
Ah. I'm using the Aggressive list which has no whitleisting I guess??

**Comment by hagezi** [comment]
Yes, wrong lsit fpr AdGuard, use: https://raw.githubusercontent.com/hagezi/dns-blocklists/main/adblock/spam-tlds.txt

**Comment by Dynamic5912** [comment]
Thank you sir!
